includes/Core/Authentication/Authentication.php

```diff
@@ -12,11 +12,12 @@
 
 use Google\Site_Kit\Context;
 use Google\Site_Kit\Core\Authentication\Clients\OAuth_Client;
+use Google\Site_Kit\Core\Authentication\Has_Connected_Admins;
+use Google\Site_Kit\Core\Authentication\Owner_ID;
```

Comment:
Nit-picking, but these are not necessary because it's the same namespace.

Reply:
Updated.